README.md

```diff
@@ -7,9 +7,19 @@ Smart contracts for C4Coin's forthcoming crowdsale
 
 ## Overview
 
-It is proposed that C4Coin will launch a [Regulation A+](https://en.wikipedia.org/wiki/Regulation_A#Regulation_A+) crowdsale, granting buyers of its tokens stock in the company.
+It is proposed that C4Coin will launch a [Regulation A+](https://en.wikipedia.org/wiki/Regulation_A#Regulation_A+) compatible token, where each issued token corresponds to a unit of stock in the company.
```

Comment:
How about: Regulation A+ "compliant" instead of "compatible"

Reply:
I have a sense this project should focus on being Delaware General Corporate Law compliant, especially in regard to registering a Regulation A+ securities offering